tools/enso4igv/pom.xml

```diff
@@ -5,7 +5,7 @@
     <artifactId>enso4igv</artifactId>
     <packaging>nbm</packaging>
     <name>Enso Language Support for NetBeans &amp; Ideal Graph Visualizer</name>
-    <version>1.13-NAPSHOT</version>
+    <version>1.15-SNAPSHOT</version>
```

Comment:
This is the _"major and minor"_ version of the `.nbm` file and `.vsix` extension. Bump it up when some important functionality changes.